build-image/first-boot.bash

```diff
@@ -2,11 +2,16 @@
 
 REPOSITORYURL=$(grep -i '^repositoryurl' /etc/openhabian.conf | cut -d '=' -f2)
```

Comment:
the strange thing about this 3 parameters is, that they are copied to openhab.conf... and the typical build-image/openhabian.conf looks quite different....
so a normal install first-boot.bash would not see any of them....

Reply:
This I was struggling with, too, and it was the reason to intro init.bash which is copying the relevant variables from there. Some of these I found to be missing and to be a problem on builds.